README.md

```diff
@@ -42,6 +42,27 @@ On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that resid
 - `mutatepods.kubemacpool.io=allocateForAll` - to opt in pods mac allocation in your namespace
 - `mutatevirtualmachines.kubemacpool.io=allocateForAll` - to opt in vms mac allocation in your namespace
 
+#### How to enable/disable kubemacpool for a namespace 
+
+Kubemacpool is disabled by default on a new namespace.
+To enable kubemacpool on a specific namespace:
+```bash
+kubectl label namespace user-namespace-opting-in-pods-vms mutatepods.kubemacpool.io=allocateForAll mutatevirtualmachines.kubemacpool.io=allocateForAll
```

Comment:
These names are too long. Would you pick something short, maybe even "example-namespace". It takes me a lot of time to go over the whole name and it does not add any value on top of the text before this code block.